source/css/bylaw.css.scss

```diff
@@ -202,3 +202,7 @@ nav.left-nav {
     }
   }
 }
+
+a {
```

Comment:
It's because I have an `a` without an `href` (it just needs to open the chat). Is there a better way?